aviary/mission/phase_builder_base.py

```diff
@@ -419,7 +419,15 @@ def _add_user_defined_constraints(self, phase, constraints):
         for constraint_name, kwargs in constraints.items():
             if kwargs['type'] == 'boundary':
                 kwargs.pop('type')
-                phase.add_boundary_constraint(constraint_name, **kwargs)
+
+                if 'target' in kwargs:
+                    # Support for constraint aliases.
+                    target = kwargs.pop('target')
+                    kwargs['constraint_name'] = constraint_name
+                    phase.add_boundary_constraint(target, **kwargs)
+                else:
+                    phase.add_boundary_constraint(constraint_name, **kwargs)
```

Comment:
would we ever want this alias for the non-boundary constraints?

Reply:
A non-boundary constraint is just a path constraint, where you specify upper or lower bounds that the whole phase shouldn't exceed. We don't support a general constraint yet (i.e., specify values at will at specific points in the trajectory), but it might be good for future work.